### ISSUE TITLE
ImageNet-22k classifier layout

### ISSUE THREAD

**Issue by rwightman** [microsoft/FocalNet#27]
Can you share the classifier layout of your ImageNet-22k heads? It does not match any canonical layouts I'm aware of and it'd be useful to have as these models are useful as is for more than just pretrain (but not without the layout).

You have 21842 classes, that does not match either the fall11 (21841) or google's internal (21843). It does not match winter21 which is < 20k classes. I cannot add a background '0' class to offset 21481 -> 21842 (assuming lexicographical sorting of the sysnets which is standard practice for imagenet classifier layouts).

Thanks

**Comment by jwyang** [comment]
Hi, @rwightman , thanks for pointing out this issue! 

Please find our 22k layout here: https://github.com/microsoft/FocalNet/blob/main/labelmap_22k_reorder.txt

We did some reorderings of the labels so that we could directly evaluate our model on in1k using the first 1000 logits. Please note that one category the 851th class (label=850) in 1k is missed in 22k. that's why we have labels in [0,21841] with 850 missed.

thanks,
Jianwei

**Comment by rwightman** [comment]
@jwyang awesome, thank you! that's definitely handy for in1k eval.
